src/discussions/data/hooks.js

```diff
@@ -167,3 +167,13 @@ export const useAlertBannerVisible = (content) => {
     || (content.abuseFlagged && canSeeReportedBanner)
   );
 };
+
+export const useIsShowLearnersTab = () => {
+  const learnersTabEnabled = useSelector(selectLearnersTabEnabled);
+  const userRoles = useSelector(selectUserRoles);
+  const isAdmin = useSelector(selectUserIsStaff);
+  const isGTA = useSelector(selectUserIsGroupTa);
+  const privileged = useSelector(selectUserHasModerationPrivileges);
+  const allowedUsers = isAdmin || isGTA || privileged || (userRoles.includes('Student') && userRoles.length > 1);
```

Comment:
Do we need the the first 3 checks here? If the userRoles includes Student and there is more than 1 role then the additional role will cover the first 3 checks here?